website/docs/advanced/strategy-constraints.md

```diff
@@ -4,7 +4,8 @@ title: Strategy Constraints
 ---
 
 :::info Availability
-Strategy constraints are available to Unleash Pro and Enterprise users.
+Strategy constraints are available to Unleash Pro and Enterprise users. 
+It is scheduled to be made available to Open Source users from **Unleash v4.16**.
```

Comment:
@thomasheartman Your input please :D

Reply:
Nice! I'd suggest just leaving this as it is for now, however: I've got a more comprehensive doc PR in #2113, which also updates notes for custom context fields (and other instances of constraints).

What do you think?

```suggestion
```

Reply:
So ideally, either merge the PRs, or revert the change you made to avoid conflicts.

Reply:
I will revert my change!